therapi-runtime-javadoc/src/main/java/com/github/therapi/runtimejavadoc/RuntimeJavadoc.java

```diff
@@ -96,7 +98,31 @@ public static ClassJavadoc getJavadoc(String qualifiedClassName, ClassLoader cla
      * @param loader             the class object to use to find the Javadoc resource file
      * @return the Javadoc of the given class
      */
-    public static ClassJavadoc getJavadoc(String qualifiedClassName, Class loader) {
+    public static ClassJavadoc getJavadoc(String qualifiedClassName, Class<?> loader) {
+        try {
+            return getSkinnyClassJavadoc(qualifiedClassName, loader)
+                    .createEnhancedClassJavadoc(loader.getClassLoader().loadClass(qualifiedClassName));
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    static ClassJavadoc getSkinnyClassJavadoc(Class<?> clazz) {
+        return getSkinnyClassJavadoc(clazz.getName(), clazz);
+    }
+
+    private static ClassJavadoc getSkinnyClassJavadoc(String qualifiedClassName, ClassLoader loader) {
+        System.out.printf("Getting %s%n", qualifiedClassName);
```

Comment:
Please remove all calls to System.out.printf

Reply:
Ah yep those should have been removed. Done